tinygrad/runtime/autogen/opencl.py

```diff
@@ -1793,3 +1793,107 @@ class struct__cl_name_version(Structure):
     'struct__cl_kernel', 'struct__cl_mem', 'struct__cl_name_version',
     'struct__cl_platform_id', 'struct__cl_program',
     'struct__cl_sampler', 'union__cl_image_desc_0']
+opencl_status_codes = {}
+opencl_status_codes[-1002] = "CL_INVALID_D3D10_DEVICE_KHR                         "
```

Comment:
all strings have a lot of spaces at the end

Reply:
fixed, but it's in the fork now that the PR is closed